### PR TITLE
ADD: beat info modal

### PR DESCRIPTION
**Pull request by co012**
Screenshot 2022-08-30 at 12 57 56
Screenshot 2022-08-30 at 12 58 42

### CODE REVIEW

File: packages/app/ui/beat-list-entry/show-more/cancelBtn.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { Pressable, Text } from "app/design-system";
+
+export function CancelBtn(props: { onPress: () => void }) {
```
Comment (jmysliv) [inline packages/app/ui/beat-list-entry/show-more/cancelBtn.tsx:3]:
```suggestion
type Props = {
    onPress: () => void;
}

export function CancelBtn({onPress}: Props) {
```

File: packages/app/ui/beat-list-entry/show-more/description.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { Image, Text, View } from "app/design-system";
+import { imageUrlMedium } from "app/utils/entry";
+
+export function Description(props: {
```
Comment (jmysliv) [inline packages/app/ui/beat-list-entry/show-more/description.tsx:4]: I'd name it differently. `Description` is not well descriptive. Maybe `BeatThumbnail` or `BeatInfo`?

File: packages/app/ui/beat-list-entry/show-more/interactions.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { Pressable, Text, View } from "app/design-system";
+import { Price } from "app/ui/beat-list-entry/price";
+import { FavoriteButton } from "app/ui/favoriteButton";
+
+export function Interactions() {
```
Comment (jmysliv) [inline packages/app/ui/beat-list-entry/show-more/interactions.tsx:5]: It's also not descriptive. What about `BeatActions`?

**Review by jmysliv — APPROVED** [review]
Looks good, just remove `@expo/vector-icons` from `package.json` if it won't be used anymore.

**Review by alejomendoza — APPROVED** [review]

**Pull request merged**